### DISCUSSION TITLE
Dns project

### DISCUSSION THREAD

**Discussion by Qazizadhhezbullah** [Diamond-Family/DNS-Installition-and-Configution#17] [General]
Hi brothers
How are you
In evening i have some problams
I have one good ideia about our project
but big problem is this we do not hava any devices

**Comment by Abdulmateenmaher** [comment]
Ok but I have an idea to solve this problem
we can implement this project on our local pcs 
using programing language as same response as DNS server scale.

**Comment by KhalilFaizi** [comment]
yes good you sign to good point
